src/app/services/poke-service.service.ts

```diff
@@ -12,14 +12,16 @@ export class PokeServiceService {
   constructor(
     private http: HttpClient
   ) { }
-
+  
+  // Gets all pokemon
   getMany() {
     let params: HttpParams = new HttpParams();
     // TODO: Get it working (likely with search bar logic) to not have to load every single pokemon, which would take a long time
     params = params.append("pageSize", 721);
     return this.http.get<PokemonList>(`${environment.apiUrl}/Pokemon`,{params});
   }
 
+  // Get an individual pokemon given its id or name
   getMon(identifier: string){
```

Comment:
The method name would be easier to work with if it's something like `getByName` or `getByIdentifier`.

It adds cognitive load to look up the method names if it's not immediately clear what they do.

I think mon may be a pokemon term but ideally the variable and method names should make sense to people messing the expert context.